package.json

```diff
@@ -77,8 +85,12 @@
     "react-admin": "^4.16.10",
     "react-dom": "^18.2.0",
     "react-idle-timer": "^5.7.2",
-    "react-scripts": "^5.0.1",
-    "typescript": "^5.3.3",
+    "typescript": "<5.4.0",
```

Comment:
I take it this is because of:
```
$ yarn lint
yarn run v1.22.22
$ eslint src
=============

WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=4.3.5 <5.4.0

YOUR TYPESCRIPT VERSION: 5.7.2

Please only submit bug reports when using the officially supported version.

=============
```
warning when using a newer typescript version.

Which is caused by old dependencies in `eslint-config-standard-with-typescript` and `eslint-plugin-vitest`. I think I used `eslint-plugin-vitest` with a very new typescript in kukkuu-ui, so it shouldn't be causing this issue. Maybe it's the `eslint-config-standard-with-typescript` as it uses older v6, eslint-plugin-vitest uses v7, but there's also v8 according to https://www.npmjs.com/package/@typescript-eslint/typescript-estree

Dependencies of `@typescript-eslint/typescript-estree`:
```
$ yarn why @typescript-eslint/typescript-estree
yarn why v1.22.22
[1/4] Why do we have the module "@typescript-eslint/typescript-estree"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "@typescript-eslint/typescript-estree@6.21.0"
info Has been hoisted to "@typescript-eslint/typescript-estree"
info Reasons this module exists
   - Hoisted from "@typescript-eslint#utils#@typescript-eslint#typescript-estree"
   - Hoisted from "eslint-config-standard-with-typescript#@typescript-eslint#parser#@typescript-eslint#typescript-estree"
   - Hoisted from "@typescript-eslint#eslint-plugin#@typescript-eslint#type-utils#@typescript-eslint#typescript-estree"
info Disk size without dependencies: "1.81MB"
info Disk size with unique dependencies: "2.73MB"
info Disk size with transitive dependencies: "4.03MB"
info Number of shared dependencies: 24
=> Found "eslint-plugin-vitest#@typescript-eslint/typescript-estree@7.18.0"
info Reasons this module exists
   - "eslint-plugin-vitest#@typescript-eslint#utils" depends on it
   - Hoisted from "eslint-plugin-vitest#@typescript-eslint#utils#@typescript-eslint#typescript-estree"
info Disk size without dependencies: "1.04MB"
info Disk size with unique dependencies: "1.95MB"
info Disk size with transitive dependencies: "3.26MB"
info Number of shared dependencies: 24
Done in 0.37s.
```

This would be good to fix without needing to limit typescript version to an older version, but I don't see this as a blocker for merging this PR. This PR is an improvement, but still, more improvements could be done, this being one of them IMHO.

Reply:
I tried again and when I upgraded those, the linting became really really really slow, so I would say that it's safer to stick with these at the moment, until we find out new configurations, taht really works.